games-fps/eduke32/eduke32-20181010.7067.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit desktop eapi7-ver gnome2-utils toolchain-funcs
+
+MY_BUILD="$(ver_cut 2)"
+MY_DATE="$(ver_cut 1)"
+
+MY_PN_HRP="duke3d_hrp"
```

Comment:
What's the purpose of all those variables?

Reply:
These are addon packages for eduke32. All are used in `SRC_URI`.
Yes, I could write MY_PN_* and MY_PV_* directly into `SRC_URI`, but I think, this makes it better to read eBuild this way.

Reply:
MY_PV might make sense but MY_PN is pretty much writing 'foo' as '${FOO}'.

Reply:
Ok, removed MY_PN, but left MY_PV.